translations/fr.po

```diff
@@ -639,18 +560,114 @@ msgstr "Paramètres"
 msgid "Donate"
 msgstr "Donner"
 
-#: ui.py:117
+#: ui.py:114
 msgid "Word Wise"
 msgstr "Word Wise"
 
-#: ui.py:120
+#: ui.py:117
 msgid " and "
 msgstr " et "
 
-#: ui.py:123
+#: ui.py:120
 msgid "Generating {} for {}"
 msgstr "Générant {} pour {}"
 
-#: ui.py:131
+#: ui.py:128
 msgid "{} generated for {}"
 msgstr "{} généré pour {}"
+
+msgid "Catalan"
+msgstr "Catalan"
+
+msgid "Danish"
+msgstr "Danois"
+
+msgid "German"
+msgstr "Allemand"
+
+msgid "Greek"
+msgstr "Grec"
+
+msgid "English"
+msgstr "Anglais"
+
+msgid "Finnish"
+msgstr "Finlandais"
+
+msgid "French"
+msgstr "Français"
+
+msgid "Serbo-Croatian"
+msgstr "Serbo-Croate"
+
+msgid "Italian"
+msgstr "Italien"
+
+msgid "Japanese"
+msgstr "Japonais"
+
+msgid "Korean"
+msgstr "Coréen"
+
+msgid "Lithuanian"
+msgstr "Lituanien"
+
+msgid "Macedonian"
+msgstr "Macédonien"
+
+msgid "Dutch"
+msgstr "Néerlandais"
+
+msgid "Norwegian Bokmål"
+msgstr "Norvégien Bokmål"
+
+msgid "Polish"
+msgstr "Polonais"
+
+msgid "Portuguese"
+msgstr "Portugais"
+
+msgid "Romanian"
+msgstr "Roumain"
+
+msgid "Russian"
+msgstr "Russe"
+
+msgid "Spanish"
+msgstr "Espagnol"
+
+msgid "Swedish"
+msgstr "Suédois"
+
+msgid "Ukrainian"
+msgstr "Ukrainien"
+
+msgid "Chinese"
+msgstr "Chinois"
+
+msgid "Choose Wiktionary language"
+msgstr "Sélectionnez la langue du Wiktionary"
+
+msgid "Customize"
+msgstr "Personaliser"
+
+msgid "Wiktionary"
+msgstr "Wiktionary"
+
+msgid "Kindle Word Wise"
+msgstr "Word Wise Kindle"
+
+msgid "Definition"
+msgstr "Définition"
+
+msgid "Forms"
+msgstr "Formes"
+
+msgid "Non-English book"
+msgstr "Livre non-Anglais"
+
+msgid ""
+"For Kindle format books, Word Wise is only available in books in English."
+msgstr ""
+"Pour les formats de livres sur Kindle. Word Wise n'est disponible que pour "
+"les livres en Anglais."
```

Comment:
These strings are no longer used and can be safely deleted.

Reply:
OK. Can you please delete them then. How would I have known?

Reply:
Sure, I'll delete those lines. Poedit comments the unused strings at the end, I guess you uncommented them to restore the language translations but also included the deleted strings. And thanks for your contribution!